pom.xml

```diff
@@ -864,17 +884,17 @@
       <dependency>
         <groupId>com.github.ben-manes.caffeine</groupId>
         <artifactId>caffeine</artifactId>
-        <version>2.3.3</version>
+        <version>2.6.1</version>
       </dependency>
       <dependency>
         <groupId>org.slf4j</groupId>
         <artifactId>slf4j-nop</artifactId>
-        <version>1.7.22</version>
+        <version>1.8.0-beta0</version>
```

Comment:
beta0? I do not think we should have any beta dependencies in the product

Reply:
I agree. That missed my initial sweep. I will update it to the latest patch`1.7.25`.